### DISCUSSION TITLE
Domain Mapping

### DISCUSSION THREAD

**Discussion by abhisaha1** [letterpad/letterpad#277] [Ideas]
Letterpad is free to use. Even the platform.  One of the most important feature would be domain mapping. This is tricky because of the SSL part. Any thoughts how we can proceed with this ?

**Comment by ASOwnerYT** [comment]
Any update on this? I want to use my Cloudflare domain with Letterpad.

**Reply by abhisaha1** [reply]
This will take a while. However, if you would like to host the client - https://github.com/letterpad/letterpad-client
You can use your own domain and still access the letterpad API.

**Comment by abhisaha1** [comment]
This is in progress and is nearing completion.

**Comment by abhisaha1** [comment]
Hidden feature. Can be accessed at https://letterpad.app/admin/domain-mapping
